spec/ndx-fiber-photometry.extensions.yaml

```
@@ -263,6 +263,11 @@ groups:
       - null
     doc: The data values. May be 1D or 2D. The first dimension must be time.The optional
       second dimension refers to the fiber that record the series.
+    attributes:
+    - name: unit
+      dtype: text
+      value: n.a.
```

How about `default_value` as in https://github.com/NeurodataWithoutBorders/nwb-schema/blob/d8d8e0332e72cd14b10bd7e809558eca31dad391/core/nwb.base.yaml#L140
```suggestion
      value: n.a.
      default_value: n.a.
```